src/test/java/net/minecraftforge/debug/DataGeneratorTest.java

```diff
@@ -136,54 +153,55 @@ protected void registerRecipes(Consumer<IFinishedRecipe> consumer)
             ResourceLocation ID = new ResourceLocation("data_gen_test", "conditional");
 
             ConditionalRecipe.builder()
```

Comment:
Maybe im reading this wrong but it seems to be a lot of unecessary line changes below, changing less lines helps better understand what you did and conserves blame.